core/storage/auth/gae_models.py

```diff
@@ -218,7 +239,7 @@ def get_by_user_id(cls, user_id):
             UserIdentifiersModel. The model with user_id field equal to user_id
             argument.
         """
-        return cls.query(cls.user_id == user_id).get()
+        return cls.query(cls.user_id == user_id).get() # type: ignore[no-any-return]
```

Comment:
Why is the ignore needed? Can you do this without an ignore? Ditto elsewhere.

Reply:
@seanlip This is required as we should return Any from get() function of the query.get() method in stubs. This is because model.query() will return the Query object. The get() function of the query object will return the Model. So in static typing I found no method to solve this issue. So the return type from the query get() and get_multi() will be `Any`. Please let me know you views.

Reply:
Why can't the return type be a union of ndb.Model or None? The returned object will be one of those, right?

Reply:
Return an object of type ndb.Model . Then mypy will not be able to detect the Type whether it is an exploration model or any other model. It will assume the type to be ndb.Model. So fields added in the model wont be accessed as mypy will assume it to be of type ndb.Model not its subclass. Thanks!

Reply:
Couldn't the stub for query.get() return (ndb.Model|None), and the caller of this method cast it to the correct type?

Reply:
This is good substitute. WIll do. Thanks!

Reply:
Succesfully changed the stubs and added casts wherever necessary. Thanks!